annotation/tests/test_jobs.py

```diff
@@ -35,15 +59,95 @@ def categories():
 
 @pytest.fixture
 def jobs_to_test_progress(job_annotators: Tuple[User], categories: Category):
-    yield Job(
-        job_id=1,
-        callback_url="http://www.test.com",
-        annotators=[job_annotators[0], job_annotators[1]],
-        validation_type=ValidationSchema.cross,
-        is_auto_distribution=False,
-        categories=categories,
-        deadline="2021-10-19T01:01:01",
-        tenant="test",
+    yield (
+        Job(
+            job_id=1,
+            name="test1",
+            callback_url="http://www.test.com",
+            annotators=[job_annotators[0], job_annotators[1]],
+            validation_type=ValidationSchema.cross,
+            is_auto_distribution=False,
+            categories=[categories],
+            deadline="2021-10-19T01:01:01",
+            tenant="test",
+        ),
+        Job(
+            job_id=2,
+            name="test2",
+            callback_url="http://www.test.com",
+            annotators=[job_annotators[0], job_annotators[1]],
+            validation_type=ValidationSchema.cross,
+            is_auto_distribution=False,
+            categories=[categories],
+            deadline="2021-10-19T01:01:01",
+            tenant="test",
+        ),
+    )
+
+
+@pytest.fixture
+def tasks():
```

Comment:
you mostly change file_id and status, could you simplify this fixture by utilizing for loops/comprehensions?